### PR TITLE
Update opendatacube.md

### PR DESCRIPTION
**Pull request by ValentinaHutter**
Update to point to openeo-processes-dask and openeo-pg-parser-networkx.

### CODE REVIEW

File: documentation/1.0/developers/backends/opendatacube.md [hunk 1]
```diff
@@ -1,224 +1,88 @@
...
-each process in the process graph is mapped separately. Next to the correct mapping of the process itself the main task
-of [openeo-odc](https://github.com/openeo-odc) is to correctly understand and pass given parameters and arguments
-(this can be simple values but also references previous nodes).
+Large data sets can be organised as `opendatacube Products` or as `STAC Collections`.
 
-::: tip Note
-opendatacube, xarray and openeo-processes-python are **not** dependencies because this package simply creates a
-python file that can be executed in the correct environment where these dependencies are resolved.
-:::
+* `opendatacube Products`: The implementation of `load_collection` can include the `opendatacube` function `datacube.Datacube.load()`. It is recommended to use the `dask_chunks` parameter, when loading the data. The function returns a `xarray DataSet`, in order to be compliant with `openeo-processes-dask`, it can be converted to a `DataArray` using the `Dataset.to_array(dim='variable', name=None)` function.
```
Comment (clausmichele) [inline documentation/1.0/developers/backends/opendatacube.md:54]: are you using `variable` in  `Dataset.to_array(dim='variable', name=None)` or `bands` at EODC?

Comment (ValentinaHutter) [reply documentation/1.0/developers/backends/opendatacube.md:54]: Oh, 'variable' is the default in the xarray documentation, we are actually using 'bands'.

**Comment by ValentinaHutter** [comment]
@clausmichele do you want to add a section about client side processing? :)

**Comment by clausmichele** [comment]
@ValentinaHutter yes as soon as I have time, it's in my to do list :)

**Comment by m-mohr** [comment]
Is this still planned to be finished?

**Comment by clausmichele** [comment]
Hi @m-mohr, yes it is planned but I totally forgot about it, I will try to update this week.

**Comment by clausmichele** [comment]
I finally found the time to do this, once Valentina merges my contribution to her fork, @m-mohr can proceed with the merge of this PR.

**Comment by clausmichele** [comment]
@m-mohr you can merge this now!

**Comment by m-mohr** [comment]
@clausmichele @ValentinaHutter You need to update the link in .vuepress/config.js. It still points to opendatacube.
Also, to keep old links active, we should add a redirect from the old location to the new location in .vuepress/enhanceApp.js

File: .vuepress/config.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ const versions = [
       {text: 'Service Providers', items: [
           {text: 'Getting Started', link: 'developers/backends/getting-started.html'},
           {text: 'Performance Guide', link: 'developers/backends/performance.html'},
-          {text: 'Open Data Cube', link: 'developers/backends/opendatacube.html'},
+          {text: 'Xarray', link: 'developers/backends/xarray.html'},
```
Comment (m-mohr) [inline .vuepress/config.js:39]:
```suggestion
          {text: 'Xarray / Dask Guide', link: 'developers/backends/xarray.html'},
```

**Comment by m-mohr** [comment]
Thank you everyone!

**Comment by ValentinaHutter** [comment]
I was just about to update the enhanceApp.js, and saw that you did that now, thank you!

**Pull request merged**